capellambse/aird/_visual.py

```diff
@@ -97,14 +103,21 @@ def shape_factory(ebd: c.ElementBuilder) -> diagram.Box:
         int(layout.attrib.get("width", "0")),
         int(layout.attrib.get("height", "0")),
     )
-    styleclass = ebd.data_element.attrib["type"]
+
+    if element is not None:
+        styleclass = "RepresentationLink"
+    else:
+        styleclass = "Note"
```

Comment:
What's going on with this part? This used to be taken from the XML, but now it's hard-coded to "Note". Is there just no other value possible here?

Reply:
Previously, the variable `styleclass` always received the value `Note` from the XML, regardless of whether it was a Note, Representation Link or Text.

The only one of those 3 that has an element attribute in the XML is Representation Link and is the only one that has slightly different styling.

I agree this is not very good and it might make sense to add something like:

```
if element is not None:
     styleclass = "RepresentationLink"
 else:
     styleclass = ebd.data_element.attrib["type"]
```
And check what other elements go into `_visual.py` and double check if everything is correct. It also makes sense to add a class for the visual element `Text` but at moment both Text and Note have the exact same styling in py-capellambse.

Reply:
> I agree this is not very good and it might make sense to add something like

We don't need that. I was just wondering because I wasn't aware that "Note" was already the only possible option to begin with. :)

LGTM.